src/viewer.js

```diff
@@ -1252,6 +1252,15 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
     },
 
 
+    //
+    /**
+     * @function
+     * @return {Boolean} returns true if the viewer is in fullscreen
```

Comment:
It looks like JSDoc allows either `@return` or `@returns` but that `@returns` is the preferred one, so please change it to that.